src/Webdam.php

```diff
@@ -83,6 +83,14 @@ public function getAsset($assetID) {
     return $this->client->getAsset($assetID);
   }
 
+  public function __call($name, $arguments) {
+    $method_variable = array($this->client, $name);
+    if(is_callable($method_variable)){
+      return call_user_func_array($method_variable, $arguments);
+    }
+  }
```

Comment:
:+1: I might go in and clean up this service a bit since this will be there now, but that can wait until after this PR lands.

Reply:
Cleanup is here: https://github.com/mobomo/media_webdam/pull/25